src/users/dto/create-user.dto.ts

```diff
@@ -0,0 +1,30 @@
+import { ApiProperty } from "@nestjs/swagger";
+import { IsEmail, IsEnum, IsNotEmpty, IsString, MinLength} from "class-validator";
+import { UserRole } from "src/constants";
```

Comment:
Deberias configurar las rutas para que no haga falta poner /src/ en tus rutas absolutas
```suggestion
import { UserRole } from "constants";
```

Reply:
Otra forma...
```suggestion
import { UserRole } from "@constants";
```